[Docs] Manual modules reload after switching DKP edition (#17289)
deckhouse/deckhouse: docs/documentation/pages/admin/configuration/registry/SWITCHING_EDITIONS.md

       type: Ready
   ```

1. Check whether there are any submodules in the `d8-*` namespaces that are in an error state and cannot load images. This must be done manually, as Deckhouse modules are not currently reinitialized automatically after the changes described above.

   Get a list of pods:

   ```shell
   d8 k get po -A
   ```

   Get detailed information about problematic submissions:

   ```shell
   d8 k describe po <pod_name> <namespace>
   ```

>    Reload the modules related to the problematic submissions by executing the following command on all master nodes:

Reviewer: ```suggestion
   Re-download the modules corresponding to the problematic pods by running the following command on all master nodes:
```

   ```shell
   rm -rf /var/lib/deckhouse/downloaded/<module-name>/
   ```

   To get `<module-name>`, run the command:

   ```shell
   d8 k get modules
   ```

>    After the data removal from all master nodes, restart Deckhouse:

Reviewer: ```suggestion
   After  removing the data of the required modules, restart Deckhouse:
```
